test/build/bare-imports/index.test.js

```diff
@@ -0,0 +1,10 @@
+const fs = require('fs');
+const assert = require('assert');
+const path = require('path');
+const execa = require('execa');
+
+it('buildOptions.metaDir', () => {
+  execa('node', ['npm', 'run', 'TEST']);
+  // expect dir in package.json to exist
+  assert(fs.existsSync(path.resolve(__dirname, 'build', 'static', 'snowpack')));
```

Comment:
ah, it's failing because the test that I gave you is bad (execa is async, not sync, so you're asserting that something exists before the build has even run).

If you rebase off of master, you should be able to copy an updated test from one of the other `build` test dirs.